qiskit/visualization/latex.py

```diff
@@ -747,18 +752,20 @@ def _build_latex_array(self):
                                         "\\ctrl{" + str(pos_2 - pos_1) + "}"
                                 self._latex[pos_2][column] = \
                                     "\\gate{R_z(%s)}" % (self.parse_params(op.op.params[0]))
-                            elif nm == "cu1":
+                            elif nm in ['cu1', 'cp']:
```

Comment:
what about `'rzz'`?

Reply:
In this never-ending 'elif', 'rzz is handled below at line 781. All of this gets cleaned up on #5855, of course. :-)